3ds/libsidplay/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: badda71 <me@badda.de>
+
+pkgname=3ds-libsidplay
+pkgver=1.36.60
+pkgrel=1.0
+pkgdesc="SID chip music module playing library"
+arch=('any')
+url="https://github.com/badda71/3ds-libsidplay"
```

Comment:
This url should ideally be the webpage of a maintained upstream if possible. It shouldn't just be a random github repo.

Reply:
actually, I have not found any maintained website. Looks like libsidplay is not maintained anymore, but I find it still very useful.

Reply:
Usefulness of libsidplay isn't really in dispute, I just don't think this URL is suitable for anyone seeking further information.

Reply:
I've changed this to the original URL via archive.org for now as per some of the other distro packages listed on https://repology.org/project/libsidplay1/packages